CRITICAL - 2024-12-06 01:36:19 --> ParseError: syntax error, unexpected variable "$producto", expecting "function" or "const"
[Method: GET, Route: productos]
in APPPATH\Controllers\ProductoController.php on line 25.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\laragon\\www\\fotobook\\app\\Controllers\\ProductoController.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\ProductoController')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\ProductoController')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\ProductoController', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-12-06 01:42:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-12-06 01:43:08 --> ErrorException: Undefined array key "nombre"
[Method: GET, Route: productos/1/edit]
in APPPATH\Views\Productos\edit.php on line 21.
 1 APPPATH\Views\Productos\edit.php(21): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "nombre"', 'C:\\laragon\\www\\fotobook\\app\\Views\\Productos\\edit.php', 21)
 2 SYSTEMPATH\View\View.php(234): include('C:\\laragon\\www\\fotobook\\app\\Views\\Productos\\edit.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('productos/edit', [], true)
 5 APPPATH\Controllers\ProductoController.php(53): view('productos/edit', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductoController->edit('1')
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductoController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-12-06 01:46:31 --> CodeIgniter\Database\Exceptions\DataException: There is no data to update.
[Method: POST, Route: productos/1/update]
in SYSTEMPATH\BaseModel.php on line 1011.
 1 SYSTEMPATH\BaseModel.php(1011): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(865): CodeIgniter\BaseModel->update([...], [])
 3 APPPATH\Controllers\ProductoController.php(65): CodeIgniter\Model->update('1', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductoController->update('1')
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductoController))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-12-06 01:55:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-06 01:56:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-06 01:56:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-06 02:04:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-06 02:23:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-06 03:06:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-06 03:13:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-12-06 03:17:25 --> ErrorException: Cannot redeclare App\Controllers\ProductoController::show()
[Method: GET, Route: productos]
in APPPATH\Controllers\ProductoController.php on line 121.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-12-06 03:17:26 --> ErrorException: Cannot redeclare App\Controllers\ProductoController::show()
[Method: GET, Route: productos]
in APPPATH\Controllers\ProductoController.php on line 121.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-12-06 03:17:26 --> ErrorException: Cannot redeclare App\Controllers\ProductoController::show()
[Method: GET, Route: productos]
in APPPATH\Controllers\ProductoController.php on line 121.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-12-06 03:17:27 --> ErrorException: Cannot redeclare App\Controllers\ProductoController::show()
[Method: GET, Route: productos]
in APPPATH\Controllers\ProductoController.php on line 121.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-12-06 03:17:59 --> ErrorException: Undefined variable $producto
[Method: GET, Route: productos/1]
in APPPATH\Views\Productos\show.php on line 14.
 1 APPPATH\Views\Productos\show.php(14): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $producto', 'C:\\laragon\\www\\fotobook\\app\\Views\\Productos\\show.php', 14)
 2 SYSTEMPATH\View\View.php(234): include('C:\\laragon\\www\\fotobook\\app\\Views\\Productos\\show.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('Productos/show', [], true)
 5 APPPATH\Controllers\ProductoController.php(28): view('Productos/show', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductoController->show('1')
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductoController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-12-06 03:19:35 --> ErrorException: Undefined array key "nombre"
[Method: GET, Route: productos/1/edit]
in APPPATH\Views\Productos\edit.php on line 22.
 1 APPPATH\Views\Productos\edit.php(22): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "nombre"', 'C:\\laragon\\www\\fotobook\\app\\Views\\Productos\\edit.php', 22)
 2 SYSTEMPATH\View\View.php(234): include('C:\\laragon\\www\\fotobook\\app\\Views\\Productos\\edit.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('productos/edit', [], true)
 5 APPPATH\Controllers\ProductoController.php(62): view('productos/edit', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductoController->edit('1')
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductoController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-12-06 03:22:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-12-06 03:27:02 --> ParseError: Unclosed '{' on line 7
[Method: GET, Route: productos/create]
in APPPATH\Controllers\ProductoController.php on line 124.
 1 SYSTEMPATH\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\laragon\\www\\fotobook\\app\\Controllers\\ProductoController.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(272): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\ProductoController')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\ProductoController')
 4 SYSTEMPATH\CodeIgniter.php(887): class_exists('\\App\\Controllers\\ProductoController', true)
 5 SYSTEMPATH\CodeIgniter.php(496): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-12-06 03:27:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-06 03:59:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-06 04:03:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-06 04:06:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-06 04:07:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-12-06 04:20:22 --> mysqli_sql_exception: Unknown column 'update_at' in 'field list' in C:\laragon\www\fotobook\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\laragon\www\fotobook\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `pr...', 0)
#1 C:\laragon\www\fotobook\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pr...')
#2 C:\laragon\www\fotobook\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pr...')
#3 C:\laragon\www\fotobook\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\laragon\www\fotobook\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\laragon\www\fotobook\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\laragon\www\fotobook\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\laragon\www\fotobook\system\BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 C:\laragon\www\fotobook\app\Controllers\ProductoController.php(69): CodeIgniter\BaseModel->save(Array)
#9 C:\laragon\www\fotobook\system\CodeIgniter.php(933): App\Controllers\ProductoController->store()
#10 C:\laragon\www\fotobook\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductoController))
#11 C:\laragon\www\fotobook\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\laragon\www\fotobook\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\laragon\www\fotobook\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\laragon\www\fotobook\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2024-12-06 04:20:22 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'update_at' in 'field list'
[Method: POST, Route: productos/store]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `productos` (`Nombre`, `Descripcion`, `Precio`, `Imagen`, `created_at`, `update_at`) VALUES (:Nombre:, :Descripcion:, :Precio:, :Imagen:, :created_at:, :update_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\ProductoController.php(69): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductoController->store()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductoController))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-12-06 04:20:22 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'update_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `productos` (`Nombre`, `Descripcion`, `Precio`, `Imagen`, `created_at`, `update_at`) VALUES (\'Teclado\', \'Gamer\', \'30\', \'1733458822_5d7d2759dec0551fa633.png\', \'2024-12-06 04:20:22\', \'2024-12-06 04:20:22\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `productos` (`Nombre`, `Descripcion`, `Precio`, `Imagen`, `created_at`, `update_at`) VALUES (\'Teclado\', \'Gamer\', \'30\', \'1733458822_5d7d2759dec0551fa633.png\', \'2024-12-06 04:20:22\', \'2024-12-06 04:20:22\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `productos` (`Nombre`, `Descripcion`, `Precio`, `Imagen`, `created_at`, `update_at`) VALUES (:Nombre:, :Descripcion:, :Precio:, :Imagen:, :created_at:, :update_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\ProductoController.php(69): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductoController->store()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductoController))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-12-06 04:20:22 --> [Caused by] mysqli_sql_exception: Unknown column 'update_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `productos` (`Nombre`, `Descripcion`, `Precio`, `Imagen`, `created_at`, `update_at`) VALUES (\'Teclado\', \'Gamer\', \'30\', \'1733458822_5d7d2759dec0551fa633.png\', \'2024-12-06 04:20:22\', \'2024-12-06 04:20:22\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `productos` (`Nombre`, `Descripcion`, `Precio`, `Imagen`, `created_at`, `update_at`) VALUES (\'Teclado\', \'Gamer\', \'30\', \'1733458822_5d7d2759dec0551fa633.png\', \'2024-12-06 04:20:22\', \'2024-12-06 04:20:22\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `productos` (`Nombre`, `Descripcion`, `Precio`, `Imagen`, `created_at`, `update_at`) VALUES (\'Teclado\', \'Gamer\', \'30\', \'1733458822_5d7d2759dec0551fa633.png\', \'2024-12-06 04:20:22\', \'2024-12-06 04:20:22\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `productos` (`Nombre`, `Descripcion`, `Precio`, `Imagen`, `created_at`, `update_at`) VALUES (:Nombre:, :Descripcion:, :Precio:, :Imagen:, :created_at:, :update_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\ProductoController.php(69): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductoController->store()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductoController))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-12-06 04:25:36 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-06 04:26:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-12-06 04:28:06 --> ErrorException: Undefined array key "usuario_id"
[Method: GET, Route: productos/12]
in APPPATH\Views\Productos\show.php on line 21.
 1 APPPATH\Views\Productos\show.php(21): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "usuario_id"', 'C:\\laragon\\www\\fotobook\\app\\Views\\Productos\\show.php', 21)
 2 SYSTEMPATH\View\View.php(234): include('C:\\laragon\\www\\fotobook\\app\\Views\\Productos\\show.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('productos/show', [], true)
 5 APPPATH\Controllers\ProductoController.php(29): view('productos/show', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductoController->show('12')
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductoController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-12-06 04:30:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-12-06 04:30:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-12-06 04:30:47 --> ErrorException: Undefined array key "usuario_id"
[Method: GET, Route: productos/1]
in APPPATH\Views\Productos\show.php on line 21.
 1 APPPATH\Views\Productos\show.php(21): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "usuario_id"', 'C:\\laragon\\www\\fotobook\\app\\Views\\Productos\\show.php', 21)
 2 SYSTEMPATH\View\View.php(234): include('C:\\laragon\\www\\fotobook\\app\\Views\\Productos\\show.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('productos/show', [], true)
 5 APPPATH\Controllers\ProductoController.php(29): view('productos/show', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\ProductoController->show('1')
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductoController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2024-12-06 04:36:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
